plugin/wavesurfer.regions.js

```diff
@@ -331,6 +331,9 @@ WaveSurfer.Region = {
             my.element.addEventListener('mousedown', onDown);
             my.wrapper.addEventListener('mouseup', onUp);
             my.wrapper.addEventListener('mousemove', onMove);
+            document.querySelector("body").addEventListener("mouseup", function () {
```

Comment:
The event handler on body should be removed on destroy. Subscribe to the wavesurfer's destroy event.